src/index.html

```diff
@@ -7,8 +7,79 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_style">
+      <div class="logo">
+        <a href="#home">
+          <img 
+          src="images/logo.png" 
+          alt="Logo" 
+          class="logo_image">
+        </a>
+      </div>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="menu-item">
+            <a href="#" class="is-active menu_link">
+              Apple
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a href="#" class="menu_link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a href="#" class="menu_link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a 
+              href="#" 
+              class="menu_link" 
+              data-qa="hover"
+            >
+              Laptops &amp; computers
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a href="#" class="menu_link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a href="#" class="menu_link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a href="#" class="menu_link">
+              Photo
+            </a>
+          </li>
+
+          <li class="menu-item">
+            <a 
+              href="#" 
+              class="menu_link"
+            >Video</a>
```

Comment:
```suggestion
            <a 
              href="#" 
              class="menu_link"
            >
              Video
            </a>
```
It's not critical, but looks better for me